pkg/config/config.go

```diff
@@ -43,8 +43,8 @@ func new() *Config {
 	conf := &Config{
 		API_SERVER_URL: getEnv("API_SERVER_URL", "https://kubernetes.default.svc"),
 		AuthCacheTTL:   getEnvAsInt("AUTH_CACHE_TTL", int(60000)),    // 1 minute
-		SharedCacheTTL: getEnvAsInt("SHARED_CACHE_TTL", int(120000)), // 2 min (increase to 10min after implementation)
-		UserCacheTTL:   getEnvAsInt("USER_CACHE_TTL", int(120000)),   // 2 min (increase to 10min after implementation)
+		SharedCacheTTL: getEnvAsInt("SHARED_CACHE_TTL", int(600000)), // 10 min
+		UserCacheTTL:   getEnvAsInt("USER_CACHE_TTL", int(600000)),   // 10 min
```

Comment:
I've been hesitant because of the risk of keeping incorrect data for too long.
Could we reduce the risk by doing 5 mins first (2.8)?  Then wait until we are very confident before changing to 10 mins in 2.9